api/server.js

```diff
@@ -26,7 +26,8 @@ var port = appConfig.get('port');        // set our port
 var routesToAuthorize = ['/api/updateapp', '/api/appprofile', 
                         '/api/uploadlogo', '/api/addapp', 
                         '/api/masterprofile', '/api/categories',
-                        '/api/approvedapps', '/api/tags', '/api/search'];
+                        '/api/approvedapps', '/api/tags',
+                        '/api/search', '/api/technologies'];
```

Comment:
no router and service defined for path 'api/technologies'. is this truly needed?

Reply:
never mind.  I saw this afterwards: https://github.com/code4romania/civichq-api/pull/87